src/capi-multiple-paidfor/web/index.scss

```diff
@@ -10,6 +10,40 @@
     color: $neutral-1;
 }
 
+.advert__title {
+  @include mq($until: tablet) {
+    width: 70%;
+    min-height: $gs-baseline * 5;
+  }
+}
+
+.badge {
+  display: inline-block;
+  @include mq($until: tablet) {
```

Comment:
hmmm ... what if the width of the badge goes beyond the 30% left? I would've preferred it if you used a flex container instead, so that space math is built in.